client/document_test.go

```diff
@@ -61,31 +75,31 @@ func TestNewFromJSON(t *testing.T) {
 	assert.Equal(t, doc.fields["Name"].Type(), LWW_REGISTER)
 	assert.Equal(t, doc.fields["Age"].Name(), "Age")
 	assert.Equal(t, doc.fields["Age"].Type(), LWW_REGISTER)
-	assert.Equal(t, doc.fields["Address"].Name(), "Address")
-	assert.Equal(t, doc.fields["Address"].Type(), OBJECT)
+	// assert.Equal(t, doc.fields["Address"].Name(), "Address")
```

Comment:
todo:  Why are these commented out? Please either document why, or put them back :)

Reply:
Because in develop it used to create a doc with a sub object which we currently don't support. I should probably just remove the commented code.

Reply:
Ah I see my bad :) Thanks for removing them